apcd-cms/src/apps/admin_regis_table/static/admin_regis_table/css/table.css

```diff
@@ -1,5 +1,6 @@
 .status-filter.org-filter {
     width: max-content;
+    padding-right: 20px;
```

Comment:
this padding allows space between the text and arrow in dropdown. With long names, it could be tight.